src/modules/cpu/kernel/brightness.hpp

```diff
@@ -216,8 +246,8 @@ RppStatus brightness_u8_u8_host_tensor(Rpp8u *srcPtr,
                         compute_brightness_16_host(p, pBrightnessParams);  // brightness adjustment
                         rpp_simd_store(rpp_store16_f32_to_u8, dstPtrTemp, p);    // simd stores
 #endif
-                        srcPtrTemp +=16;
-                        dstPtrTemp +=16;
+                        srcPtrTemp +=max_length;
```

Comment:
Please add space after += operator
Applicable to all such instances in the file